lib/offsite_payments/integrations/payu_in.rb

```diff
@@ -70,6 +70,7 @@ def initialize(order, account, options = {})
           super
           @options = options
           self.pg = 'CC'
+          @fields['udf5'] = 'shopify_active_merchants'
```

Comment:
`ActiveMerchant` is used by many people outside of shopify. How will this impact these people?

Reply:
We already have the `application_id` field that can be set by applications and then read by integrations:
https://github.com/activemerchant/offsite_payments/blob/34059fdc120509583112f4c3c02496c4cc39517c/lib/offsite_payments/helper.rb#L9-L12